.github/workflows/sync-docs-from-node.yml

```
@@ -128,776 +116,570 @@ jobs:
+      - name: Check for changes and create branch
+        id: check_changes
         run: |
-          set -euo pipefail
-          git push --force-with-lease origin ${{ env.BRANCH_NAME }}
-
-      - name: Capture timestamp
-        id: timestamp
-        run: echo "utc=$(date -u +"%Y-%m-%dT%H:%M:%SZ")" >> "$GITHUB_OUTPUT"
-
+          source .github/scripts/git-utils.sh
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)